components/PayPalButton.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <div>
+    <q-spinner
+      v-if="loading" />
+    <q-banner
+      v-if="error"
+      class="text-white bg-red">
+      We are terribly sorry, but PayPal is not available
+      at this time.
+    </q-banner>
+    <div
+      id="paypal-button" />
+  </div>
+</template>
+
+<script>
+/*!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+/**
+ * PayPalButton.vue is a simple wrapper around paypal's smart buttons.
+ * You can login into https://www.sandbox.paypal.com with your sandbox
+ * account to see the transactions in development mode.
+ */
+export default {
+  name: 'PayPalButton',
+  props: {
+    // paypal clientId
+    clientId: {
+      type: String,
+      required: true
+    },
+    // This determines if we are making a one time charge
+    // or a recurring subscription charge.
+    subscription: {
+      type: Boolean,
+      default: false
+    },
+    disableCard: {
+      type: Array,
+      default: () => ['amex']
```

Comment:
I actually agree this with this. I put `amex` in there, because my PayPal account is not a pro-account and can not accept it yet. Addressed here: https://github.com/digitalbazaar/bedrock-web-payment/pull/1/commits/ad2507115de81dcf96f940e9ce7172f59760240a